README.md

```diff
@@ -1 +1,179 @@
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Technology Innovation Summit 2023] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Technology Innovation Summit 2023]** is an online website for a technology conference with the big figures of the industry.
+
+> Video description: [@loomhandle](https://www.loom.com/share/216a834a707246da9fdccf8ffa452918)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack">HTML</a>
+
+### Tech Stack <a name="tech-stack">CSS</a>
+
+### Tech Stack <a name="tech-stack">JavaScript</a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Innovation Technology Summit 2023]** Show informations about a technology conference that will take place soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Portfolio](https://tshobohwa.github.io/Technology-Innovation-Summit-2023/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Code Editor
+- Browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- cd my-folder
+- git clone "https://github.com/Tshobohwa/Technology-Innovation-Summit-2023"
+- cd Portfolio
+
+### Install
+
+Install this project with:
+
+- cd my-project
+- gem install
+
+### Usage
+
+To run the project, execute the following command:
+
+- live server
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Tshobohwa)
+- Twitter: [@twitterhandle](https://twitter.com/SalomonTshoboh1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tshobohwa-salomon-4bb457245/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [Program] **[new_feature_1]**
+- [Join] **[new_feature_1]**
+- [Sponsor] **[new_feature_1]**
+- [New] **[new_feature_1]**
+
```

Comment:
- [ ] Kindly after including the future features, you can remove the  `new_feature_1` as it is only supposed to guide you.